tachyon/zk/plonk/permutation/permutation_argument.h

```diff
@@ -35,36 +35,36 @@ class TACHYON_EXPORT PermutationArgument {
   // The argument may use larger degree gates depending on the actual
   // circuit's degree and how many columns are involved in the permutation.
   size_t RequiredDegree() const {
-    // degree 2:
-    // l_first(X) * (1 - z(X)) = 0
+    // See https://zcash.github.io/halo2/design/proving-system/permutation.html
+    // for more details.
     //
-    // We will fit as many polynomials pᵢ(X) as possible
-    // into the required degree of the circuit, so the
-    // following will not affect the required degree of
-    // this middleware.
+    // degree 2: l_first(X) * (1 - Z(X)) = 0
```

Comment:
Either put the operations on the next row, or change every other comment regarding `degree [#]`to also be in-line